jbpm-human-task/jbpm-human-task-workitems/src/test/java/org/jbpm/services/task/wih/util/HumanTaskHandlerHelperTest.java

```diff
@@ -1710,6 +1710,46 @@ public void testNotStartedNotifyMinimalWithHtmlWithISOExpirationTimePeriodFormat
 
 		assertEquals(4, roundExpirationTime(expirationTime));
 	}
+	
+	
+    @Test
+    public void testNotStartedNotifyMinimalWithBodyWithNewLineWithISOExpirationTimePeriodFormat() {
+        WorkItem workItem = new WorkItemImpl();
+        workItem.setParameter("NotStartedNotify", "[tousers:john|subject:Test of notification|body:viva er Beti\nabajo el Sevilla]@[PT4H]");
```

Comment:
Sample valid but not politically correct. 
```suggestion
        workItem.setParameter("NotStartedNotify", "[tousers:john|subject:Test of notification|body:viva er Beti\nmanque pierda]@[PT4H]");
```